src/shared/components/query/QueryStore.ts

```diff
@@ -477,6 +479,14 @@ export class QueryStore
 		}
 	});
 
+	readonly molecularProfilesInSelectedStudies = remoteData<MolecularProfile[]>({
+		invoke: async()=>{
+			const profiles:CacheData<MolecularProfile[], string>[] =
+				await this.molecularProfilesInStudyCache.getPromise(this.selectedStudyIds, true);
+			return _.flatten(profiles.map(d=>(d.data ? d.data : [])));

```

Comment:
what's reason that data would not be present?

Reply:
if theres an error in the cache loading then data would be null